RatLib/GameWindowLocator.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Drawing;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RatLib
+{
+	public static class GameWindowLocator
```

Comment:
This is the class I'm using to find the Tarkov main window and get its size. I also have a helper function to access the settings files for Tarkov, but its not being used right now. I left it in because it could be useful in the future to determine some other things.